fjord/base/models.py

```diff
@@ -3,21 +3,14 @@
 from django.contrib.auth.models import User
 from django.db import models
 
-import caching.base
 from tower import ugettext_lazy as _lazy
 
 from fjord.base import forms
 from fjord.base.validators import EnhancedURLValidator
 
 
-class ModelBase(caching.base.CachingMixin, models.Model):
-    """Common base model for all models: Implements caching."""
-
-    objects = caching.base.CachingManager()
-    uncached = models.Manager()
-
-    class Meta:
-        abstract = True
+# Common base model for all fjord models
+ModelBase = models.Model
```

Comment:
I left this instead of changing imports everywhere. But I could go either way. I have no idea if we'll need a common base class in the future vs the builtin django models.Model.